lib/ui/painting.dart

```diff
@@ -860,6 +860,64 @@ class Path extends NativeFieldWrapperClass2 {
   void _arcTo(double left, double top, double right, double bottom,
               double startAngle, double sweepAngle, bool forceMoveTo) native "Path_arcTo";
 
+  /// Appends up to four conic curves weighted to describe an oval of [radius]
+  /// and rotated by [xAxisRotation].
+  ///
+  /// The first curve begins from the last point in the path and the last ends
+  /// at [arcEnd]. The curves follow a path in a direction determined by
+  /// [isClockwiseDirection] and [isLargeArc] in such a way that the sweep angle
+  /// is always less than 360 degrees.
+  ///
+  /// A simple line is appended if either either radii are zero or the last
+  /// point in the path is [arcEnd]. The radii are scaled to fit the last path
+  /// point if both are greater than zero but too small to describe an arc.
+  ///
+  void arcToPoint(Offset arcEnd, {
+      Radius radius : Radius.zero,
+      double xAxisRotation : 0.0,
+      bool isLargeArc : false,
+      bool isClockwiseDirection : true,
+    }) {
+    assert(_offsetIsValid(arcEnd));
+    _arcToPoint(arcEnd.dx, arcEnd.dy, radius.x, radius.y, xAxisRotation,
+                isLargeArc, isClockwiseDirection);
+  }
+
+  void _arcToPoint(double arcEndX, double arcEndY, double radiusX,
```

Comment:
nit: to match prevailing style, no blank line between these two

Reply:
Done.